inngest-spring-boot-adapter/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ repositories {
 }
 
 dependencies {
-    val pkg = if (System.getenv("RELEASE") != null) "com.inngest:inngest:[0.0.5, 0.1)" else project(":inngest")
+    val pkg = if (System.getenv("RELEASE") != null) "com.inngest:inngest:[0.0.5, 0.1]" else project(":inngest")
```

Comment:
hm I may have botched this range, supposedly `[0.0.5, 0.1]` excludes version `0.1.0`. This probably should have been `[0.0.5, 0.1.0]` or even `[0.1.0, 0.2.0)`. Rather than trying to fix it, we're just rolling forward with all the 0.2.0 upgrades in
https://github.com/inngest/inngest-kt/pull/90
https://github.com/inngest/inngest-kt/pull/91